### PR TITLE
spruce up README

### PR DESCRIPTION
**Pull request by technicalpickles**
This is all boiler plate from the generator. I wanted to give a little clarity about what this particular repository is, and also just remove all the stuff that isn't relevant unless you are setting up the app initially.

As part of this, I added a `bin/setup-heroku` to automate an operator being able to do Heroku things. I also saw that we're using heroku redis brain, but didn't have a Heroku addon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
-deployed on [Heroku][heroku] to get you up and running as quick as possible.
-
-This README is intended to help get you started. Definitely update and improve
-to talk about your own instance, how to use and deploy, what functionality is
-available, etc!
-
-[heroku]: http://www.heroku.com
-[hubot]: http://hubot.github.com
-[generator-hubot]: https://github.com/github/generator-hubot
+hubot is a chat bot built on the [Hubot][hubot] framework for helping the [Hubot community on Slack](https://hubot-slackin.herokuapp.com/).
```
Comment (bkeepers) [inline README.md]: the `[hubot]` link isn't defined.  Also there are a lot of `hubot`s in that sentence. 

> This is an instance of [Hubot][hubot] for helping the [Hubot community on Slack](https://hubot-slackin.herokuapp.com/).

File: README.md [hunk 2]
```diff
@@ -34,26 +22,6 @@ Then you can interact with hubot by typing `hubot help`.
...
-    grep -o 'hubot-[a-z0-9_-]\+' external-scripts.json | \
-      xargs -n1 -I {} sh -c 'sed -n "/^# Configuration/,/^#$/ s/^/{} /p" \
-          $(find node_modules/{}/ -name "*.coffee")' | \
-        awk -F '#' '{ printf "%-25s %s\n", $1, $2 }'
-
-How to set environment variables will be specific to your operating system.
-Rather than recreate the various methods and best practices in achieving this,
-it's suggested that you search for a dedicated guide focused on your OS.
-
 ### Scripting
```
Comment (bkeepers) [inline README.md]: Is this section necessary? I suspect anyone making changes to this repo will be involved in the Hubot community and know where to find docs.

**Review by bkeepers — APPROVED** [review]
A few minor comments, but otherwise 👍

File: README.md (hunk 2 above)
Comment (technicalpickles) [inline README.md]: It doesn't hurt, but I will tweak a little. Our internal hubot still links to this scripting reference.

**Pull request merged**